applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_incompressible_potential_element.cpp

```diff
@@ -129,6 +151,47 @@ namespace Kratos {
       KRATOS_CHECK_NEAR(RHS(5), -0.5, 1e-7);
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(EmbeddedIncompressiblePotentialFlowElementCalculateLocalSystem, CompressiblePotentialApplicationFastSuite)
+    {
+      Model this_model;
+      ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+      GenerateEmbeddedElement(model_part);
+      Element::Pointer pElement = model_part.pGetElement(1);
+
+
+      // Define the nodal values
+      Vector potential(3);
+      potential(0) = 1.0;
+      potential(1) = 2.0;
+      potential(2) = 3.0;
+
+      // Define the distance values
+      Vector level_set(3);
+      level_set(0) = 1.0;
+      level_set(1) = -1.0;
+      level_set(2) = -1.0;
+
+      for (unsigned int i = 0; i < 3; i++){
+        pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY_POTENTIAL) = potential(i);
+        pElement->GetGeometry()[i].FastGetSolutionStepValue(LEVEL_SET) = level_set(i);
+      }
+
+      // Compute RHS and LHS
+      Vector RHS = ZeroVector(3);
+      Matrix LHS = ZeroMatrix(3, 3);
+
+      pElement -> Set(BOUNDARY);
+      pElement->CalculateLocalSystem(LHS, RHS, model_part.GetProcessInfo());
+
+      // Check the RHS values (the RHS is computed as the LHS x previous_solution,
+      // hence, it is assumed that if the RHS is correct, the LHS is correct as well)
+      KRATOS_CHECK_NEAR(RHS(0), 0.125, 1e-7);
```

Comment:
Save all the expected values in an `std::vector<double>` an then loop it. In this way you avoid to have three `KRATOS_CHECK_NEAR` macros. The point of doing this is that this kind of macros increase quite a lot the compilation time.

Reply:
Ok! Thanks for the suggestion! I also updated the rest of the tests in the same file.